### ISSUE TITLE
pLogs wont load?

### ISSUE THREAD

**Issue by Hellathicc** [aStonedPenguin/pLogs-2#5]
Hello, I do not mean to be a nuisance, however, pLogs just does not seem to want to load. I get no LUA errors, but whenever I type /logs or !logs (I made it logs in config), it doesnt open. I am the correct rank that I set into the config as well, so I have no idea what the hell is going on...

**Comment by mark27205** [comment]
Yeah, same here, maybe I will make an Pull request when I have time to fix this.
